include/osquery/status.h

```diff
@@ -33,13 +33,14 @@ namespace osquery {
 
 class Status {
  public:
+  static constexpr int success_code = 0;
```

Comment:
Should this start with a k? and then be in all uppercase?

Reply:
Oh, you are right! Thanks!